micrometer-core/src/main/java/io/micrometer/core/instrument/dropwizard/DropwizardFunctionTimer.java

```diff
@@ -104,7 +110,7 @@ public long getMax() {
                     @Override
                     public double getMean() {
                         double count = count();
-                        return count == 0 ? 0 : totalTime(baseTimeUnit()) / count;
+                        return count == 0 ? 0 : totalTime(TimeUnit.NANOSECONDS) / count;
                     }
```

Comment:
You might want to add a comment for why the hardcoded base unit is needed

Reply:
@checketts Thanks for the feedback! I added a comment in c478cc41c419c5edf06c2e9c220467c48646a308 as suggested.